### ISSUE TITLE
Thetvapp

### ISSUE THREAD

**Issue by Shoninzo** [mrgsi/repository.glive#4]
Thetvapp addon isn't working

**Comment by mrgsi** [comment]
Yes, it's broken. I believe the website was changed so the add-on needs to be re-coded. It could be the human verification system they built into the webpages. Not something I am knowledgeable enough to fix. unfortunately. I will keep an eye out for an upgraded version

**Comment by Shoninzo** [comment]
I guess you haven't had any luck with finding an updated version yet.

**Comment by mrgsi** [comment]
Nope, no luck yet. It may not be an easy fix.

**Comment by Shoninzo** [comment]
They removed the authentication

**Comment by Shoninzo** [comment]
It's working again

**Comment by pal1000** [comment]
FYI I found a solution to overcome Cloudflare bot protection while it was still in place. For thetvapp.to [sending the web requests to origin server IP directly did the trick](https://scrapeops.io/web-scraping-playbook/how-to-bypass-cloudflare/#option-1-send-requests-to-origin-server), but only over HTTPS which required disabling TLS, validation, because if I was downgrading to HTTP, the server was able catch my trick and was sending me back through Cloudflare. I was able to find the origin server IP by using either [Censys](https://search.censys.io/search?resource=hosts&sort=RELEVANCE&per_page=25&virtual_hosts=EXCLUDE&q=thetvapp.to) or [Shodan](https://www.shodan.io/search?query=thetvapp.to). Censys search results include the subdomains and 1 erroneous result.
